examples/python/signed_distance_fields/main.py

```diff
@@ -101,35 +88,40 @@ def log_mesh(path: Path, mesh: Trimesh) -> None:
     # that `mesh_to_sdf` returns.
     bs1 = mesh.bounding_sphere
     bs2 = mesh_to_sdf.scale_to_unit_sphere(mesh).bounding_sphere
-    mesh_format = get_mesh_format(mesh)
 
     scale = bs2.scale / bs1.scale
     center = bs2.center - bs1.center * scale
-    rr.log_mesh_file(
-        "world/mesh",
-        mesh_format,
-        mesh_path=path,
-        transform=np.array([[scale, 0, 0, center[0]], [0, scale, 0, center[1]], [0, 0, scale, center[2]]]),
-    )
+
+    mesh3d = rr.Asset3D.from_file(path=str(path))
```

Comment:
Not how I have to cast to `str` :grimacing:

Reply:
Also note how everything is both awful and inconsistent here.

This shouldn't be `Asset3D.from_file`, this should be extra args to `Asset3D.__init__`.

Reply:
https://github.com/rerun-io/rerun/issues/3555